### PR TITLE
Add migratus to the template

### PR DESCRIPTION
**Pull request by g-krisztian**

### CODE REVIEW

File: resources/leiningen/new/xiana/test/app_name_fixture.clj [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  (let [config (config/edn)
+        system (-> config
+                   embedded-postgres!
+                   migrate!
+                   {{sanitized-name}}/system
+                   component/start)]
+    (try
+      (f)
+      (finally
+        (component/stop system)))))
```
Comment (defalut) [inline resources/leiningen/new/xiana/test/app_name_fixture.clj:43]: regarding DB, this calls `(dissoc this :datasource :connection)`
Does `EmbeddedPostgres`'s `.close` happen somewhere?

File: resources/leiningen/new/xiana/test/app_name_test.clj [hunk 2]
```diff
@@ -2,22 +2,11 @@
   (:require
     [clj-http.client :as http]
     [clojure.test :refer [deftest is use-fixtures]]
-    [com.stuartsierra.component :as component]
     [{{sanitized-name}}]
-    [framework.config.core :as config]))
+    [{{sanitized-name}}-fixture :refer [std-system-fixture]]))
```
Comment (defalut) [inline resources/leiningen/new/xiana/test/app_name_test.clj:6]: With more fixtures and test helpers added this ns might be renamed to something like test-utils and eventually moved to the framework itself, for now I am fine

File: resources/leiningen/new/xiana/project.clj [hunk 3]
```diff
@@ -44,6 +46,6 @@
                        :migration-dir "migrations"
                        :db            {:classname   "com.mysql.jdbc.Driver"
                                        :subprotocol "postgres"
-                                       :subname     "//localhost/controllers"
+                                       :subname     "//localhost:5433/{{sanitized-name}}"
```
Comment (defalut) [inline resources/leiningen/new/xiana/project.clj:49]: Is it possible to load the config for lein migrate from config.edn?
There is >2y unresolved issue on that here https://github.com/yogthos/migratus-lein/issues/24

**Review by defalut — APPROVED** [review]
Ensure `EmbeddedPostgres`'s `.close` is called

File: resources/leiningen/new/xiana/test/app_name_fixture.clj (hunk 1 above)
Comment (g-krisztian) [reply resources/leiningen/new/xiana/test/app_name_fixture.clj:43]: > regarding DB, this calls `(dissoc this :datasource :connection)`
> Does `EmbeddedPostgres`'s `.close` happen somewhere?

Not directly, going to add it. Maybe the JVM will kill it anyway.

File: resources/leiningen/new/xiana/project.clj (hunk 3 above)
Comment (g-krisztian) [reply resources/leiningen/new/xiana/project.clj:49]: It's only for migratus plugin `lein migratus migrate`

**Review by brahayan-dev — APPROVED** [review]

**Pull request merged**